main_routers/live2d_router.py

```
@@ -0,0 +1,887 @@
+        return {"success": True, "message": "参数保存成功"}
+    except Exception as e:
+        logger.error(f"保存模型参数失败: {e}")
+        return JSONResponse(status_code=500, content={"success": False, "error": str(e)})
+
+
+@router.get('/live2d/load_model_parameters/{model_name}')
+def load_model_parameters(model_name: str):
+    """从模型目录的parameters.json文件加载参数"""
+    try:
+        # 查找模型目录
+        model_dir, url_prefix = find_model_directory(model_name)
```

Ruff (0.14.8)

Unpacked variable `url_prefix` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)